proto/api/v1/summary_service.proto

```diff
@@ -17,6 +17,7 @@ message SummaryCountsResponse {
   int64 num_nodes = 6;
 }
 
+// TODO(ROX-24528): This API is deprecated, remove it in 4.7
 service SummaryService {
```

Comment:
Comments from this file are rendered in the built-in API so they are user facing. Hence please also add a comment for every `rpc` call in this service.

Reply:
Yeah I'd rather we leave the comment at the top (not as a description of the service / or an API call) so that an internal JIRA number isn't rendered within our API documentation.

Reply:
I've moved the comment to the top of the file. I didn't see it rendered before though, is [this](https://staging.demo.stackrox.com/main/apidocs#tag/SummaryService/operation/SummaryService_GetSummaryCounts) the built-in API you are referring to?

Reply:
Yes.